src/components/SignUp/index.js

```diff
@@ -1,19 +1,65 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+import { useMutation } from '@apollo/react-hooks'
+import gql from 'graphql-tag'
+
 import styles from './SignUp.module.css'
+import { Form, Button, Jumbotron, Container, Alert } from 'react-bootstrap'
 
-import Form from 'react-bootstrap/Form'
-import Button from 'react-bootstrap/Button'
-import Jumbotron from 'react-bootstrap/Jumbotron'
-import Container from 'react-bootstrap/Container'
-import { Link } from 'react-router-dom'
+const SIGN_UP = gql`
+  mutation sign_up($firstname:String!, $lastname:String!, $username: String!, $password: String!){
+    signup(data:{firstName:$firstname,lastName:$lastname,username:$username,password:$password}){
+      user{
+        id
+        username
+      }
+      jwt
+    }
+  }
+`
 
 function SignUp () {
+  const [show, setShow] = useState(false)
+  const [alertInfo, setAlertInfo] = useState({ variant: 'danger', message: 'Unknown error' })
+
+  const successfulSignup = ({ signup }) => {
+    setAlertInfo({ variant: 'success', message: 'Singed up succesfully' })
+    setShow(true)
+    console.log('success signup', signup)
```

Comment:
Console logging is for development purposes only. Remove them all before pushing the code to be reviewed.